carma_types_mod.F90

```diff
@@ -509,6 +509,18 @@ module carma_types_mod
     real(kind=f), allocatable, dimension(:)          :: f_wave       ! (NWAVE)
     real(kind=f), allocatable, dimension(:)          :: f_dwave      ! (NWAVE)
     logical, allocatable, dimension(:)               :: f_do_wave_emit  ! (NWAVE)
+
+    !  Sulfate Nucleation Methods:
```

Comment:
How is this being used? Does Vehkamaki have heterogeneous nucleation? What would you set this for if you wanted Zhao for heterogenous and Vehkamki for homogeneous? Is this really a specifier for the homogeneous nucleation method and not the heterogenous nucleation method? It might be good to rename this variable or at least add comments, if it only controls the homogeneous nucleation.